contents/docs/integrate/_snippets/install-react.mdx

```diff
@@ -15,7 +15,7 @@ VITE_PUBLIC_POSTHOG_KEY=<ph_project_api_key>
 VITE_PUBLIC_POSTHOG_HOST=<ph_client_api_host>
 ```
 
-3. Integrate PostHog at the root of your app (such as `main.jsx` if you are using Vite).
+3. Integrate PostHog at the root of your app (such as `main.jsx` if you are using Vite). Setting `capture_pageview` to `history_change` is recommended as React usually acts as a single-page app and doesn't fire page load events on route changes.
```

Comment:
This is only true if they're using a router layer - which is usually true, but still, we could mention they don't need this in case they're building a basic React app.

Reply:
But then, of course, this won't hurt then, so this might be fine.

Reply:
I do say usually in this sentence